addons/project/views/project_portal_views.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="portal_view_task_kanban" model="ir.ui.view">
```

Comment:
guidelines : id would be "portal_task_view_kanban" (module_model_view/action_viewType)
You can also adapt all the other record id of the file